test/label.util.test.ts

```diff
@@ -226,7 +240,8 @@ test('control with label object, text-only and required', t => {
     } as JsonSchema;
     const controlElement = {type: 'ControlElement', scope: {$ref: '#/properties/foo'},
       label: {text: 'mega bar'}} as ControlElement;
-    const labelObject = getElementLabelObject(schema, controlElement);
+    const labelObject = getElementLabelObject({schema: schema, dropPoints: {}} as ItemModel,
```

Comment:
Extract creation of `ItemModel` into helper factory function, which makes it easy to update the initialization of the item. Also removes the necessity to cast.